IndicoV2/IndicoClient.cs

```diff
@@ -56,7 +63,38 @@ public IndicoClient(string apiToken, Uri baseUri, bool verify = true)
             _apiToken = apiToken ?? throw new ArgumentNullException(nameof(apiToken));
             BaseUri = baseUri ?? throw new ArgumentNullException(nameof(baseUri));
             _verifySsl = verify;
-            HttpClient = new HttpClient(new AuthenticatingMessageHandler(baseUri, apiToken));
+            var handler = new AuthenticatingMessageHandler(baseUri, apiToken);
+            HttpClient = new HttpClient(handler);
+            var options = new GraphQLHttpClientOptions
+            {
+                EndPoint = new Uri($"{baseUri}/graph/api/graphql"),
+                HttpMessageHandler = handler
+            };
+            GraphQLHttpClient = new GraphQLHttpClient(options, new NewtonsoftJsonSerializer(), HttpClient);
+        }
+
+        /// <summary>
+        /// Create a new GraphQL request
+        /// </summary>
+        /// <returns>GraphQLRequest</returns>
+        public GraphQLRequest.GraphQLRequest GraphQLRequest(string query=null, string operationName=null, object variables = null)
+        {
+            var request = new GraphQLRequest.GraphQLRequest(GraphQLHttpClient);
```

Comment:
what do we expect to happen if `query` or `operationName` are null?

Reply:
if query is null it should throw a HttpRequestException. I believe operationName is more of like reference/metadata, so if operationName is null, the call still goes through assuming a valid query and variables. Let me know if I'm wrong here

Reply:
You're right. Though, we can raise an exception earlier if the query is null and explicitly say that the users need to define a query or mutation. In the doc string, it would be useful to point out that `query` is the grapqhl query or mutation, and `variables` are the variables defined in the query. If `operationName` is entirely optional, it might not even be useful to expose it here.

Reply:
removed operationName as it is optional. added param definitions to doc string and now throwing `GraphQLException` when query is `null`